purchases_ui_flutter/pubspec.yaml

```diff
@@ -11,7 +11,10 @@ environment:
   flutter: '>=3.3.0'
 
 dependencies:
-  purchases_flutter: 6.17.0
+  # We can't depend on exactly the same version
+  # because it won't be available when bumping the version
+  # until it's released.
+  purchases_flutter: ^6.0.0
```

Comment:
@tonidero this wasn't working on this branch because `6.18.0` isn't available yet. By using this, we allow it to resolve whenever it's released.
Does that make sense to you?

Reply:
I'm holding releasing this until you give it the green light

Reply:
Ohh right... Hmm so I'm wondering it's true that this branch would be in an invalid state until the deployment is done... But is that causing errors in the tests? I'm just wondering if we are running tests against the latest deployed version of the library instead of the local version in the repo...

If that's not the case, I think this should be safe to release yeah.

Reply:
> Hmm so I'm wondering it's true that this branch would be in an invalid state until the deployment is done

That's correct

> But is that causing errors in the tests? I'm just wondering if we are running tests against the latest deployed version of the library instead of the local version in the repo..

Well, we run `flutter pub get` and other commands, and because `purchases_ui_flutter` on the release branch (like this one) depends on a yet-unpublished version of `purchases_flutter`, it fails to download dependencies.

> I'm just wondering if we are running tests against the latest deployed version of the library instead of the local version in the repo

I guess that'd be true for running `purchases_ui_flutter` tests, since that depends on `purchases_flutter`'s version and not the local copy :/

Reply:
Yeah I think we can go with that for now... we should just be aware that tests using purchases_ui_flutter might be using the latest stable version, and not the branch version of purchases_flutter.

Reply:
Talked about this offline. We decided that this is good for now. If it causes issues we can investigate further.